external-import/misp/docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
       - MISP_ENFORCE_WARNING_LIST=false # Optional, enforce warning list in MISP queries
       - MISP_REPORT_TYPE=misp-event # Optional, report_class if creating report for event
       - MISP_IMPORT_FROM_DATE=2000-01-01 # Required, import all event from this date
-      - MISP_IMPORT_TAGS=opencti:import,type:osint # Optional, list of tags used for import events
+      - MISP_IMPORT_TAGS= # Optional, list of tags used to filter events to import
```

Comment:
Hi @romain-filigran,
Can you also apply this change to the `config.yml.sample` file ? 
I've also looked at the readme but it doesn't seem to be ambiguous.

Reply:
You're right. `config.yml.sample` updated also.